app/controllers/games_controller.rb

```diff
@@ -38,6 +39,50 @@ def update
     handle_update_errors
   end
 
+  def forfeit
+    if current_user.id == current_game.white_user_id
+      current_game.update_attributes(game_winner: current_game.black_user_id)
+    else
```

Comment:
I would explicitly check for `elsif current_user.id == current_game.black_user_id` so that a logged-in user who is neither white nor black cannot set white as the winner.